views/entity.html

```diff
@@ -16,7 +16,8 @@
 
     {% include "includes/list/errors.html" %}
 
-    <h1 class="govuk-heading-xl">Tell us about the overseas entity</h1>
+    <span class="govuk-caption-xl govuk-!-padding-bottom-1">{{ entityName }}</span>
```

Comment:
Isn't the story asking for the text to say "Overseas Entity <NAME>"? Because in the example they gave they said they entered "Name test" rather than "Overseas Entity Name test"

Reply:
Thanks Sam, you are right it needs to be prefixed by `Overseas Entity`.

Reply:
Will do